CedarJava/src/test/java/com/cedarpolicy/PolicyFormatterTests.java

```diff
@@ -0,0 +1,50 @@
+package com.cedarpolicy;
+
+import com.cedarpolicy.formatter.PolicyFormatter;
+import com.cedarpolicy.model.exception.InternalException;
+import org.junit.jupiter.api.Test;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+public class PolicyFormatterTests {
+
+  @Test
+  public void testPoliciesStrToPretty() throws Exception {
```

Comment:
Nit: I'd prefer that test policies (original and formatted) are stored in files, which makes it much easier to update them.

Reply:
Sounds good, I've updated the tests to use the resource files.